docs/src/main/paradox/scala/http/release-notes.md

```diff
@@ -24,7 +24,43 @@ From now on, content types missing a charset can be both parsed and expressed di
 
 *Note to scala users*: If you have `match` statements across `ContentType`, keep an eye out for new compiler hints. You need
 to decide what what to do in case you get a content type with a missing character set, by adding a 
-`ContentType.WithMissingCharset` case. 
+`ContentType.WithMissingCharset` case.
+
+### Server-Sent Events Support
+
+Support for Server-Sent events was added by merging akka-sse project by [@hseeberger](https://github.com/hseeberger). Kudos to you, Heiko!
+
+### List of Changes
+
+#### Improvements
+
+##### akka-http-core
+
+* HTTPS proxy support ([#192](https://github.com/akka/akka-http/issues/192))
+* Allow '=' in query param values in relaxed mode ([#1120](https://github.com/akka/akka-http/issues/1120))
+
+##### akka-http
+
+* Add support for Server-Sent Events ([#669](https://github.com/akka/akka-http/issues/669))
+* Add support for textual content types with missing character set ([#1134](https://github.com/akka/akka-http/issues/1134))
+
+##### akka-http-testkit
+
+* Remove unnecessary dependency to ScalaTest from JUnitSuiteLike ([#1147](https://github.com/akka/akka-http/issues/1147))
+
+##### Documentation
+
+* Document pluggable client transport infrastructure and HTTPS proxy support ([#192](https://github.com/akka/akka-http/issues/192))
+* Reference security announcements and release notes in ToC ([#1199](https://github.com/akka/akka-http/issues/1199))
+
+#### Bug Fixes
+
+##### akka-http-core
+
+* Parse Websocket headers according to the set header processing mode ([#1166](https://github.com/akka/akka-http/issues/1166))
+* Fix `idle-timeout` setting [#1012](https://github.com/akka/akka-http/issues/1012))
+* Add specific handling for `charset` param when comparing MediaTypes ([#1139](https://github.com/akka/akka-http/issues/1139))
```

Comment:
Special handle charset parameter in Accept headers, since when comparing media types? If I get the description of the issue correctly?